core-bundle/src/Resources/contao/library/Contao/Image.php

```diff
@@ -843,10 +843,10 @@ private static function getImageSizeConfig($size)
 		}
 
 		$imageSize = new \stdClass();
-		$imageSize->width = $predefinedSizes[$size]['width'];
-		$imageSize->height = $predefinedSizes[$size]['height'];
-		$imageSize->resizeMode = $predefinedSizes[$size]['resizeMode'];
-		$imageSize->zoom = $predefinedSizes[$size]['zoom'];
+		$imageSize->width = $predefinedSizes[$size]['width'] ?? 0;
+		$imageSize->height = $predefinedSizes[$size]['height'] ?? 0;
+		$imageSize->resizeMode = $predefinedSizes[$size]['resizeMode'] ?? 'proportional';
```

Comment:
```suggestion
		$imageSize->resizeMode = $predefinedSizes[$size]['resizeMode'] ?? 'crop';
```
The default is `crop` I think.